02- Running ADO Scanner from command line/Readme.md

```diff
@@ -121,3 +121,13 @@ The Get-AzSKADOSecurityStatus command now supports checkpointing via a "-UsePart
 ```PowerShell
 Get-AzSKADOSecurityStatus-OrganizationName "<OrganizationName>" -ScanAllResources -UsePartialCommits
 ```
+
+----------------------------------------------
+
+### Execute SVTs using "-UseGraphAccess" switch
+
+Some AZSK.ADO controls require graph access for correct evaluation and by using a special flag "-UseGraphAccess" in the scan command which can be used to fetch the graph access token for the user. This switch is “off by default” and control results for controls that depend on AAD group expansion may not be accurate.
```

Comment:
A**z**SK.ADO 
Some AzSK.ADO controls require graph access for correct evaluation. To fetch the graph access token for the user a special flag "-UseGraphAccess" should be used in the scan command. This switch is “off by default” and control results for such controls that depend on AAD group expansion may not be accurate.

Reply:
changes done